_studio/shared/umc/codec/jpeg_common/include/bitstreamin.h

```diff
@@ -36,7 +37,7 @@ class CBitStreamInput
   CBitStreamInput(void);
   virtual ~CBitStreamInput(void);
 
-  JERRCODE Attach(CBaseStreamInput* in);
+  JERRCODE Attach(std::shared_ptr<CBaseStreamInput> in);
```

Comment:
this will call heavy shared_ptr copy constructor. I don't think that this is desired